src/preset_cli/cli/superset/import_.py

```diff
@@ -79,57 +83,39 @@ def import_ownership(
     """
     Import resource ownership from a YAML file.
     """
-    auth = ctx.obj["AUTH"]
-    url = URL(ctx.obj["INSTANCE"])
-    client = SupersetClient(url, auth)
-
-    logs = get_logs()
-    failed_assets = (
-        {log["uuid"] for log in logs["assets"] if log["status"] == "FAILED"}
-        if logs.get("assets")
-        else set()
-    )
+    client = SupersetClient(baseurl=URL(ctx.obj["INSTANCE"]), auth=ctx.obj["AUTH"])
```

Comment:
I was getting a `too-many-locals` complain, so did this to avoid adding the skip check